vowpalwabbit/cbify.cc

```diff
@@ -40,6 +40,8 @@ struct cbify
   MwtExplorer<example>* mwt_explorer;
   vw_recorder* recorder;
   v_array<action_score> a_s;
+  // used as the seed
+  size_t example_counter;
```

Comment:
I joined the dark side by relying on calloc. John laughed pretty hard on me when he saw me doing it…